test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/ArrayModelBinderTest.cs

```diff
@@ -25,10 +26,10 @@ public class ArrayModelBinderTest
             var binder = new ArrayModelBinder<int>();
 
             // Act
-            var result = await binder.BindModelAsync(bindingContext);
+            var result = await binder.BindModelResultAsync(bindingContext);
 
             // Assert
-            Assert.NotEqual(ModelBindingResult.NoResult, result);
+            Assert.NotEqual(default(ModelBindingResult), result);
```

Comment:
Might be worth reintroducing `ModelBindingResult.NoResult` for tests only. Of course you've done the work already...